metadata/src/lib.rs

```diff
@@ -401,6 +404,14 @@ impl StorageMetadata {
         self.entries.values()
     }
 
+    /// Filter out storage entries that are not of interest.
+    pub fn retain_entries<F>(&mut self, f: F)
```

Comment:
Is this actually used anywhere? I think it can be deleted?

Reply:
removed